scripts/cargo-build.sh

```diff
@@ -1,7 +1,18 @@
 #!/usr/bin/env bash
 
-# TODO: Check for valid JSON in ALL_PROPOSALS_PARAMETERS_JSON ?
-
 export WASM_BUILD_TOOLCHAIN=nightly-2021-02-20
 
-cargo +nightly-2021-02-20 build --release
\ No newline at end of file
+FEATURES=
+if [[ "$RUNTIME_PROFILE" == "TESTING" ]]; then
+  FEATURES="testing_runtime"
+fi
+
+if [[ "$RUNTIME_PROFILE" == "STAGING" ]]; then
+  FEATURES="staging_runtime"
+fi
+
+if [[ "$RUNTIME_PROFILE" == "PLAYGROUND" ]]; then
+  FEATURES="playground_runtime"
+fi
```

Comment:
Is it worth adding to `run-dev-chain.sh` script too?
If so, perhaps it's worth refactoring those settings to a separate script? (since they're also used by `/devops/git-hooks/pre-push`)

I suggest to make the profile configurable for `start.sh` also.
It is currently only possible to use `production` profile for `start.sh`, because the env is not passed to `runtime-code-shasum.sh`.

Reply:
Well actually for start.sh you can run with different profiles because `scripts/runtime-code-shasum.sh` does use the RUNTIME_PROFILE from env.
So you can do:

```
export RUNTIME_PROFILE=TESTING
./build-node-docker.sh
./start.sh
```

Reply:
factored our the FEATURES assignment into a separate `features.sh` that is `source`ed from the bash scripts that need it. Done in 9cba30c3aeaaf273331ed4535e4b09baf0d91ac7